CONTRIBUTING.md

```diff
@@ -4,16 +4,19 @@ Building Cabal for hacking
 --------------------------
 
 The current recommended way of developing Cabal is to use the
-`v2-build` feature which [shipped in cabal-install-1.24](http://blog.ezyang.com/2016/05/announcing-cabal-new-build-nix-style-local-builds/).  Assuming
-that you have a sufficiently recent cabal-install (see above),
-it is sufficient to run:
+`v2-build` feature which [shipped in cabal-install-1.24](http://blog.ezyang.com/2016/05/announcing-cabal-new-build-nix-style-local-builds/).  If you use the latest stable version of cabal, it is sufficient to run:
 
 ```
 cabal v2-build cabal
 ```
 
-To build a local, development copy of cabal-install.  The location
-of your build products will vary depending on which version of
+If you use an older cabal, you aren't able build the testsuite, so you need to disable the default `cabal.project` that includes the testsuite, e.g.:
```

Comment:
What is "an older cabal"? I think that I have the latest release, installed by GHCup. Does it mean "a cabal older than the one in this repository"?

Assuming my interpretation is correct, then I'd find this easier to understand:
```suggestion
Versions of `cabal-install` that are older than the one in this repository, including the latest release, can't build the test suite. In those situations, disable the default `cabal.project` that includes the testsuite, e.g.:
```

Reply:
No. I wasn't precise enough. I meant "latest stable version on Hackage" and you, first, missed the previous sentence, second, appear to care about the latest stable version on ghcup, which is older. I will try to fix this.

Reply:
How about now?

Reply:
This is great, it makes perfect sense now. Thank you!

I hadn't actually realized that there were Hackage releases not in ghcup - I should write more Haskell code than I do!